plugins/asus-hid/fu-asus-hid-device.c

```diff
@@ -78,10 +78,17 @@ fu_asus_hid_device_init_seq(FuAsusHidDevice *self, GError **error)
 static gboolean
 fu_asus_hid_device_probe(FuDevice *device, GError **error)
 {
-	if (fu_device_has_flag(device, FWUPD_DEVICE_FLAG_IS_BOOTLOADER))
-		fu_hid_device_set_interface(FU_HID_DEVICE(device), 0);
-	else
-		fu_hid_device_set_interface(FU_HID_DEVICE(device), 2);
+	FuAsusHidDevice *self = FU_ASUS_HID_DEVICE(device);
+
+	fu_hid_device_set_interface(FU_HID_DEVICE(device), 0);
+
+	for (guint i = 0; i < self->num_mcu; i++) {
+		g_autoptr(FuDevice) dev_tmp =
+		    fu_asus_hid_child_device_new(fu_device_get_context(device), i);
```

Comment:
if you pass the `proxy` rather then the context the ctx gets set automatically.

Reply:
ack, switched